easybuild/easyconfigs/l/libmcfp/libmcfp-1.3.4-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
+easyblock = 'CMakeMake'
+
+name = 'libmcfp'
+version = '1.3.4'
+
+homepage = 'https://github.com/mhekkel/libmcfp'
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+description = """A library for parsing command line arguments and configuration files and making
+them available throughout a program."""
+
+github_account = 'mhekkel'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['79783923527711bc27219592f25b9e92d6e402e15be146eeb6cb2050796ec3b6']
+
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('CMake', '3.29.3'),
+    ('Catch2', '2.13.10'),
+]
+
+configopts = '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON '
+configopts += '-DFETCHCONTENT_FULLY_DISCONNECTED=ON '
+
+runtest = True
+
+sanity_check_paths = {
+    'files': [],
```

Comment:
I assume this produces an actual library then that should be in files for verification.